engine/classes/ElggInstaller.php

```diff
@@ -1635,9 +1632,9 @@ protected function createSessionFromFile() {
 	 */
 	protected function createSessionFromDatabase() {
 		$app = $this->getApp();
-		$session = ElggSession::fromDatabase($app->_services->config, $app->_services->db);
+		$session = ElggSession::fromDatabase($app->internal_services->config, $app->internal_services->db);
 		$session->start();
-		$app->_services->setValue('session', $session);
-		_elgg_services()->dic->set('session', $session);
+		$app->internal_services->set('session', $session);
+		_elgg_services()->set('session', $session);
```

Comment:
setting the same value twice. Shouldn't this be done in both internal an public service?